commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPFriendlyURLEntryLocalServiceImpl.java

```diff
@@ -322,13 +322,8 @@ protected void validate(
 		}
 
 		if (classPK > 0) {
-			CPFriendlyURLEntry cpFriendlyURLEntry =
-				cpFriendlyURLEntryPersistence.fetchByG_C_C_L_U(
-					groupId, classNameId, classPK, languageId, urlTitle);
-
-			if (cpFriendlyURLEntry != null) {
-				return;
-			}
+			cpFriendlyURLEntryPersistence.fetchByG_C_C_L_U(
+				groupId, classNameId, classPK, languageId, urlTitle);
```

Comment:
@marco-leo just want to mark here again since the previous PR was closed. I think we need to revisit this spot and see what it's supposed to be doing.